Firestore/core/test/unit/api/load_bundle_task_test.cc

```diff
@@ -26,6 +26,20 @@
 namespace firebase {
 namespace firestore {
 namespace api {
+namespace {
+
+std::unique_ptr<util::Executor> CreateUserQueue() {
+  return util::Executor::CreateSerial("Testing User Queue");
+}
+
+class LoadBundleTaskTest : public ::testing::Test {
+ public:
+  LoadBundleTaskTest() : task(CreateUserQueue()) {
+  }
+
+ protected:
+  LoadBundleTask task;
```

Comment:
Optional: you could move the initializer here and remove the default constructor if you prefer:
```cpp
class LoadBundleTaskTest : public ::testing::Test {
 protected:
  LoadBundleTask task = CreateUserQueue();
  // Or
  LoadBundleTask task = util::Executor::CreateSerial("Testing User Queue");
};
```

Reply:
This does not work because
 `Executor` cannot be directly assigned to task. I think I will just keep it as-is.

Reply:
Ah, if it has an explicit constructor, than you need to use direct initialization:
```cpp
protected:
  LoadBundleTask task{CreateUserQueue()};
```
Fine to just leave as is.